### ISSUE TITLE
Trigger icon buggy during quantize ?

### ISSUE THREAD

**Issue by bltzr** [ossia/score#1278]

**Comment by jcelerier** [comment]
What does link mean ?

**Comment by bltzr** [comment]
Blink, it does mean blink

**Comment by bltzr** [comment]
blame my bépo keyboard

**Comment by jcelerier** [comment]
ahaha makes more sense, thanks

**Comment by jcelerier** [comment]
hmm but this is already what is happening ? are you not seeing that on your side ? 

https://user-images.githubusercontent.com/2772730/130266784-16dbe96a-1311-4199-9d70-4c6f21e676ce.mp4

**Comment by bltzr** [comment]
not really… (actually it was because it got selected when clicking, but there seems also to be a graphic bug)

https://user-images.githubusercontent.com/113982/130311127-a6b243dc-b19a-413f-a586-5fe9d0728c89.mov

**Comment by jcelerier** [comment]
wut XD

**Comment by jcelerier** [comment]
arg, I tried on mac and linux without and with retina and I cannot reproduce it. Does it happen on all your computers ? is it on a M1 (I really doubt it's because of that but at this point I don't have a clue)

**Comment by jcelerier** [comment]
problem is due to hi-dpi handling, we must recompute the pixmaps when the DPI of the window changes but that is not really doable in Qt 5.15 AFAIK
